src/Vector2_TEST.py

```diff
@@ -0,0 +1,262 @@
+# 
+  # Copyright (C) 2021 Open Source Robotics Foundation
+  #  
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #  
+  #       http://www.apache.org/licenses/LICENSE-2.0
+  #  
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+  #
+#
+
+import unittest
+import math
+from ignition.math import Vector2d
+
+class Vector2Test(unittest.TestCase):
+
+  def test_construction(self):
+    v = Vector2d()
+    self.assertAlmostEqual(0.0, v.X())
+    self.assertAlmostEqual(0.0, v.Y())
+
+    vec = Vector2d(1, 0)
+    self.assertEqual(vec.X(), 1)
+    self.assertEqual(vec.Y(), 0)
+
+    # Copy constructor
+    vec2 = Vector2d(vec)
+    self.assertEqual(vec2, vec)
+
+    # Copy operator
+    vec3 = vec
```

Comment:
I think that both copy constructor and copy operator make sense only under C++ world.

Reply:
Addressed